streams/src/test/java/org/apache/kafka/streams/processor/internals/ProcessorContextImplTest.java

```diff
@@ -98,12 +123,15 @@ public void setup() {
             mock(ThreadCache.class)
         );
 
-        context.setCurrentNode(new ProcessorNode<String, Long>("fake", null, emptySet()));
+        context.setCurrentNode(new ProcessorNode<String, Long>("fake", null,
+            new HashSet<>(asList("LocalKeyValueStore", "LocalWindowStore", "LocalSessionStore"))));
     }
 
     @Test
-    public void testKeyValueStore() {
-        doTest("KeyValueStore", (Consumer<KeyValueStore<String, Long>>) store -> {
+    public void testGlobalKeyValueStore() {
```

Comment:
Can we improve the test names? `globalKeyValueStoreShouldBeReadOnly()` -- similar other methods